operations/mimir/ruler-remote-evaluation.libsonnet

```
@@ -33,7 +33,11 @@
     $.querier_args +
     $.querierUseQuerySchedulerArgs(rulerQuerySchedulerName) + {
       'querier.max-concurrent': $._config.ruler_querier_max_concurrency,
-    },
+    } + if !useRulerQueryFrontend then {} else {
+    // The ruler-querier-frontend send a query request to the ruler-query
+    'server.grpc-max-send-msg-size-bytes': $._config.ruler_remote_evaluation_max_query_response_size_bytes,
```

This should be this flag I believe:

```suggestion
    'querier.frontend-client.grpc-max-send-msg-size': $._config.ruler_remote_evaluation_max_query_response_size_bytes,
```